docs/site/themes/template/layouts/partials/download-table.html

```diff
@@ -52,4 +52,9 @@ <h3>{{ .version }}</h3>
             </tbody>
         </table>
     {{ end }}
-{{ end }}
\ No newline at end of file
+    {{ if .ova }}
+        <p>Open virtual Appliance (OVA) 
+        <a href="{{ .ova_page_link }}" title="download page"</a> for this release of Tanzu Community Edition
```

Comment:
Is this a generic link for all download? This page loops through all releases, so the `ova_page_link` will end up being the same for each entry.

The netlify jobs fail here with:
```
12:09:40 AM: Error: Error building site: failed to render pages: render of "section" failed: "/opt/build/repo/docs/site/themes/template/layouts/_default/download.html:14:15": execute of template failed: template: _default/download.html:14:15: executing "main" at <partial "download-table.html" .>: error calling partial: execute of template failed: html/template:partials/download-table.html: "<" in attribute name: "</a> for this release of Tanzu C"
```

Not sure what that is, but it doesn't like this format. I get the same running locally - did this work for you?

Reply:
Ah, the `<a>` tags are mangled. The opening `<a>` never closes. I think what you want (assuming the yaml formatting gets fixed) is:

```html
    {{ if .ova }}
        <p>Open virtual Appliance (OVA)
        <a href="{{ .ova.ova_page_link }}">download page</a> for this release of Tanzu Community Edition
        </p>
    {{ end }}
```

This runs fine locally for me.

But also, capitalize the "V" in "Virtual" and end with a period.

Reply:
Success now, and I added "vSphere" into the line.